SDWebImage/Core/SDImageCodersManager.m

```diff
@@ -36,6 +37,9 @@ - (instancetype)init {
     if (self = [super init]) {
         // initialize with default coders
         _imageCoders = [NSMutableArray arrayWithArray:@[[SDImageIOCoder sharedCoder], [SDImageGIFCoder sharedCoder], [SDImageAPNGCoder sharedCoder]]];
+        if (@available(iOS 11, macOS 10.14, tvOS 11, watchOS 4, *)) {
```

Comment:
This need to be re-considerated.

Current HEIC Animated decoding speed is dramatically slow on both iOS && macOS. Even you have HEVC hardware codec. I assume this is Apple's issue. I've fired a radar.

Maybe make it configurable and not the default value is a better way. And to say, there are nearlly no any animated HEIC image on the hole Internet.